### DISCUSSION TITLE
Why CLI does not list total number of errors if apply or apply-unsafe option is on?

### DISCUSSION THREAD

**Discussion by vestimir** [biomejs/biome#1578] [Q&A] [answered]
When I run `biome lint --max-diagnostics=5000 .` I get:

```
Checked 546 file(s) in 53ms
Found 167 error(s)
```

When I run `biome lint --max-diagnostics=5000 --apply .` I get:

```
Fixed 546 file(s) in 90ms
```

But no stats on the remaining errors. Why is that?

**Comment by castarco** [comment]
Hi @vestimir . What was the number of remaining errors? I suppose that it would be normal to say nothing if it was 0, so I'm curious if that wasn't the case.

**Comment by vestimir** [comment, marked as answer]
Unfortunately I didn't save the branch to have a reproducable case. What fixed the issue back then for me was upgrading to the latest Biome and making sure my config has the same schema version.
